web/templates/note/form.html.eex

```diff
@@ -0,0 +1,9 @@
+<%= form_for @changeset, @action, fn f -> %>
+<div>
```

Comment:
This should all be indented so that you can visually see that it's part of the outer form tag